src/app/signers/SignersMap.tsx

```diff
@@ -0,0 +1,213 @@
+          return australiaCenterAndZoom;
+        case Continent.SouthAmerica:
+          return southAmericaCenterAndZoom;
+        case Continent.Asia:
+          return asiaCenterAndZoom;
+        default:
+          return { center: [20, 0], zoom: 1 };
+      }
+    },
+    [
+      northAmericaCenterAndZoom,
+      africaCenterAndZoom,
+      europeCenterAndZoom,
+      australiaCenterAndZoom,
+      southAmericaCenterAndZoom,
+      asiaCenterAndZoom,
+    ]
+  );
+
+  useEffect(() => {
+    const { center, zoom } = getContinetCenterAndZoom(activeContinent);
+    map.setView(center, zoom);
+  }, [map, activeContinent, getContinetCenterAndZoom]);
+
+  return null;
+};
+
+const continentBoundaries: Record<
+  Continent,
+  { minLat: number; maxLat: number; minLng: number; maxLng: number }
+> = {
+  Africa: {
+    minLat: -34.833333,
+    maxLat: 37.348222,
+    minLng: -17.625,
+    maxLng: 51.272,
+  },
+  Asia: {
+    minLat: -10.0,
+    maxLat: 81.858711,
+    minLng: 26.0,
+    maxLng: 180.0,
+  },
+  Europe: {
+    minLat: 34.0,
+    maxLat: 71.0,
+    minLng: -25.0,
+    maxLng: 40.0,
+  },
+  'North America': {
+    minLat: 5.493,
+    maxLat: 83.6341,
+    minLng: -168.1195,
+    maxLng: -52.233,
+  },
+  'South America': {
+    minLat: -56.05,
+    maxLat: 13.5,
+    minLng: -81.73,
+    maxLng: -34.58,
+  },
+  Australia: {
+    minLat: -47.0,
+    maxLat: -10.0,
+    minLng: 112.0,
+    maxLng: 179.0,
+  },
+  // Antarctica: {
+  //   minLat: -90.0,
+  //   maxLat: -60.0,
+  //   minLng: -180.0,
+  //   maxLng: 180.0,
+  // },
+};
+
+export function getContinent(latitude: number, longitude: number) {
+  for (const [continent, bounds] of Object.entries(continentBoundaries)) {
+    if (
+      latitude >= bounds.minLat &&
+      latitude <= bounds.maxLat &&
+      longitude >= bounds.minLng &&
+      longitude <= bounds.maxLng
+    ) {
+      return continent as Continent;
+    }
+  }
+  return undefined;
+}
+
+export function SignersMap({
+  signersLocation,
+  activeContinent,
+}: {
+  signersLocation: {
+    lat: number;
+    lng: number;
+  }[];
+  activeContinent: Continent | null;
+}) {
+  return (
+    <MapContainer
+      center={[0, 0]}
+      zoom={1}
+      minZoom={1}
+      maxZoom={4}
+      scrollWheelZoom={false}
+      doubleClickZoom={false}
+      zoomControl={false}
+      dragging={false}
+      attributionControl={false}
+      className="roberto"
+      style={{
+        height: '100%',
+        width: '100%',
+        backgroundColor: 'var(--stacks-colors-slate-100)',
+        borderRadius: '0.75rem',
+      }}
+      bounds={[
+        [85, -180],
+        [-85, 180],
+      ]}
+      maxBoundsViscosity={1.0}
+      maxBounds={[
+        [85, -180],
+        [-85, 180],
+      ]}
+    >
+      <TileLayer
+        // url="https://{s}.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}{r}.png"
+        // url="https://{s}.basemaps.cartocdn.com/rastertiles/light_nolabels/{z}/{x}/{y}{r}.png"
```

Comment:
Leaving these as the mapbox needs an api key and if we need to quickly switch to another basemap because we have reached the mapbox free request threshold